README.md

```diff
@@ -329,14 +329,21 @@ cause the application to `abort()` on programming errors that would be logged
 and ignored at runtime.
 
 
-### Skipping the valgrind tests
-
-If you are trying to iterate quickly, you can temporarily skip the valgrind
-memory tests by running the test suite with:
+### Running tests with valgrind
+Assuming your current working directory is `debugbuild` as described above:
 ```
-MMD_SKIP_VALGRIND=True ninja test
+meson test --suite=ci_valgrind \
+  --wrap="valgrind \
+          --leak-check=full \
+          --suppressions=/usr/share/glib-2.0/valgrind/glib.supp \
+          --suppressions=../contrib/valgrind/libmodulemd-python.supp
```

Comment:
```suggestion
          --suppressions=../contrib/valgrind/libmodulemd-python.supp"
```

The closing double quote went missing again. Assuming CI passes, everything else with this PR looks great! Thanks!